Concepts docs and new FCL Quickstart (#323)
onflow/flow: docs/content/flow-port/index.md

---
title: Flow Port

> description: How to use the Flow Port

Reviewer: ```suggestion
description: How to use Flow Port
```

---

Welcome to Flow Port, your portal to the decentralized world of Flow. Access your Flow account, interact with the blockchain, manage your assets and more.
## Creating an Account

> In order to access Flow Port, you must have a valid Flow address. If you do not have a Flow address, you can choose to create one via any of our FCL Wallet Providers.

Reviewer: ```suggestion
In order to access Flow Port, you must have a valid Flow address. If you do not have a Flow address, you can choose to create one via any of the available FCL wallet providers.
```

#### Navigate to Flow Port to Create an Address

 1. Navigate to [Flow Port](https://port.onflow.org/).

 2. Click on 'Sign Up' if you need to create a new Flow Account.

 3. Click on Ledger.

 4. Follow the prompts on the screen. Plug in your Ledger device and open the Flow App.

 5. Click on Create an account. Follow the prompts on your Ledger device.

 6. Once your account address is created, you will be automatically logged into Flow Port.

## Staking & Delegating

> For a detailed walkthrough on how to use the Flow Port for staking and delegating, please read the [Flow Port staking walkthrough](/flow-port/staking-guide)

Reviewer: ```suggestion
For a detailed walkthrough on how to use Flow Port for staking and delegating, please read the [Flow Port staking walkthrough](/flow-port/staking-guide)
```
